README.md

```diff
@@ -1,2 +1,25 @@
 # java-multiaddr
-Java implementation of multiaddr
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-multiformats-blue.svg?style=flat-square)](http://github.com/multiformats/multiformats)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+
+> Java implementation of [multiaddr](https://github.com/multiformats/multiaddr).
+
+This is a **WIP**.
```

Comment:
Nah, I made this before I saw the rest of the work you ported over. Assumed it was going to be an empty README. May have jumped the README gun. :D